pkg/validation/image.go

```diff
@@ -0,0 +1,53 @@
+package validation
+
+import (
+	"fmt"
+	"image/png"
+	"io"
+	"os"
+)
+
+const (
+	kbInByte    = 1000
+	sizeLimitKB = 100
+	maxW        = 512
+	maxH        = 512
+	minW        = 128
+	minH        = 128
+)
+
+func ValidatePngImageDimension(file io.Reader) error {
+	img, err := png.DecodeConfig(file)
+	if err != nil {
+		return fmt.Errorf("failed to decode image: %w", err)
+	}
+
+	if img.Width > maxW || img.Height > maxH || img.Height < minH || img.Width < minW || img.Width != img.Height {
+		return fmt.Errorf("%w: logo should be 256x256: given %dx%d", ErrInvalidImgDimension, img.Width, img.Height)
+	}
```

Comment:
@vikmeup do we really support size 128x128? I guess it's not required

Reply:
Error sais that logo should be 256x256, although it's flexible

Reply:
I took this logic from the old bot, looks like ideally we need 256x256, but if it's less or bigger a bit - it's okay

Reply:
256 by 256 is ideal or higher, but 128 min is fine for the tokens too.

but it make sense to pass size var into the error message for consistent, avoid hard coding numbers